test-ciphers.sh

```diff
@@ -1,56 +1,74 @@
 #!/usr/bin/env bash
 
-CONSOLE_URL=$(oc get console.config.openshift.io cluster --template '{{.status.consoleURL}}')
-if [ -z "${CONSOLE_URL}" ]
-then
-  echo "no console_url, are you connected to a cluster?"
-  exit 1
+set -o errexit
+set -o nounset
+set -o pipefail
+
+oc project openshift-console
+
+CONSOLEPOD=$(oc get pod -l app=console -o jsonpath="{.items[0].metadata.name}" -n openshift-console)
+CONSOLE_POD_IP=$(oc get pod "${CONSOLEPOD}" --template '{{.status.podIP}}')
+
+echo "pod ${CONSOLEPOD} at ${CONSOLE_POD_IP}"
+
+POD_NAME="test-console-ciphers"
+CONFIGMAP_NAME="${POD_NAME}"
+FILE_NAME="test-ciphers-from-pod.sh"
+FILE_PATH="/scripts"
+
+# TODO: delete this
+oc delete pod "${POD_NAME}"
+oc delete configmap "${CONFIGMAP_NAME}"
+
+# put the test script in a configmap in the console namespace
+oc create configmap "${CONFIGMAP_NAME}" --from-file="./test/${FILE_NAME}"
+
+# creates a POD to run the test script in the console namespace
+# passes $CONSOLE_POD_IP as $SERVER environment variable
+cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: openshift-console-test
+  name: "${POD_NAME}"
+spec:
+  restartPolicy: Never
+  containers:
+  - name: "${POD_NAME}"
+    image: quay.io/benjaminapetersen/console-ciphertest:latest
```

Comment:
@benjaminapetersen You should be able to use our builder image, which already has openssl

```suggestion
    image: quay.io/coreos/tectonic-console-builder:v19
```